lib/src/controls/inputs/toggle_switch.dart

```diff
@@ -345,7 +347,7 @@ class ToggleSwitchThemeData with Diagnosticable {
                   style,
                   states,
                 ).basedOnLuminance()
-              : style.inactiveColor,
+              : style.brightness.isLight ? const Color(0xFF5A5A5A) : const Color(0xFFCCCCCC),
```

Comment:
Make color customisable in `ToggleSwitchThemeData` or `ThemeData` depending on their use.